src/explore-education-statistics-admin/src/components/editable/EditablePageModeToggle.tsx

```diff
@@ -1,73 +1,85 @@
 import { EditingMode, useEditingContext } from '@admin/contexts/EditingContext';
+import styles from '@admin/components/editable/EditablePageModeToggle.module.scss';
 import FormRadioGroup from '@common/components/form/FormRadioGroup';
+import { useMobileMedia } from '@common/hooks/useMedia';
 import useToggle from '@common/hooks/useToggle';
 import React from 'react';
 import classNames from 'classnames';
-import styles from './EditablePageModeToggle.module.scss';
 
 interface Props {
   canUpdateRelease?: boolean;
   previewLabel?: string;
   showTablePreviewOption?: boolean;
 }
-const EditablePageModeToggle = ({
+
+export default function EditablePageModeToggle({
   canUpdateRelease = true,
   previewLabel = 'Preview content',
   showTablePreviewOption = false,
-}: Props) => {
+}: Props) {
   const { editingMode, setEditingMode } = useEditingContext();
   const [isOpen, toggleOpen] = useToggle(true);
+  const { isMedia: isMobileMedia } = useMobileMedia();
```

Comment:
I am not sure making this work on mobile was necessary, as it's an Admin page.

Reply:
We generally try to keep it useable on mobile (which includes small tablets).